cli/utils_s390x.go

```diff
@@ -0,0 +1,12 @@
+// +build s390x
```

Comment:
This is technically redundant as the filename already ends in the build architecture.

Reply:
you're right. I remove it